back/engines/free/user_confirmation/app/interactors/user_confirmation/schedule_code_expiration.rb

```diff
@@ -0,0 +1,14 @@
+module UserConfirmation
+  class ScheduleCodeExpiration < ApplicationInteractor
+    delegate :user, to: :context
+
+    def call
+      schedule_code_expiration_job
+    end
+
+    def schedule_code_expiration_job
+      expire_at = user.email_confirmation_code_sent_at || Time.zone.now + 1.day
```

Comment:
Does `||` take precedence over `+`? Otherwise, it would look to me that confirmation codes expire right away, but I guess that won't be the case :p

Reply:
Good catch too 👍